src/static/css/main.css

```diff
@@ -1,2 +1,5 @@
 @import "reset.css";
 @import "link.css";
+@import "color.css";
```

Comment:
Order is important here— you'll want to start with `reset`, then `variables`, then the rest.